index.bs

```diff
@@ -3122,25 +3131,25 @@ Each {{WebRequestEvent}} has:
           1. Switch on |body|'s [=body/source=]:
 
               : [=byte sequence=]
-              :: [=list/Append=] a new {{UploadData}} with {{UploadData/bytes}}
+              :: [=list/Append=] an {{UploadData}} with {{UploadData/bytes}}
```

Comment:
nit: empty?

Reply:
Not empty because we're setting its values immediately after. I don't want to use [=new=] either though because it's not a new JS object. I'm going to use this pattern for now and see how Reilly/Dominic feel about it.